adoc/chapters/cuda_backend.adoc

```diff
@@ -0,0 +1,598 @@
+| [code]#unsampled_image_accessor<T, 2, Mode, image_target::device># | sycl::cuda::surface<T, 2> | Convert a SYCL `unsampled_image_accessor` to the `sycl::cuda::surface` interoperability type with the same type and dimensions.
+| [code]#unsampled_image_accessor<T, 3, Mode, image_target::device># | sycl::cuda::surface<T, 3> | Convert a SYCL `unsampled_image_accessor` to the `sycl::cuda::surface` interoperability type with the same type and dimensions.
+|====
+
+[[sec:cuda_support_of_core_features]]
+== CUDA Support of Core SYCL Features
+
+Some core SYCL features require a minimum compute capability for the CUDA
+backend.
+
+[[table.extensionsupport]]
+.CUDA support for Core SYCL API features
+[width="100%",options="header",cols="33%,33%,33%"]
+|====
+| Feature                                   | SYCL Aspect               | Required Compute Capability 
+| [code]#16-bit floating point#             | [code]#aspect::fp16#      | 5.3 or greater
+|====
+
+[[sec:non_core_features_and_extensions]]
+== Non-core features and extensions
+
+Some additional functions are provided for the CUDA backend in the
+`sycl::cuda` namespace.
+
+[[table.noncorefeatures]]
+.CUDA support for non-Core SYCL APIs
+[width="100%",options="header",cols="33%,33%,33%"]
+|====
+| API                                                    | Description
+| [code]#bool sycl::cuda::has_native_event(sycl::event)# | Returns `true` if the SYCL event has a valid `CUevent` that can be queries via application interop.
+|====
+
+Additional CUDA features are available depending upon the device's compute 
+capability. SYCL can support these optional CUDA features with extensions.
+
+Use of CUDA extensions requires that the API for a given extension is available
+to the SYCL implementation. This needs to be determined at compile time. 
+Checking for the existence of feature test macros is the preferred method
+for checking whether an API exists. The feature test macro format 
+is `SYCL_EXT_<vendor>_<feature>`. The `<vendor>` string may also contain the
+word `CUDA` for features specific to CUDA. For example, the feature test macro
+for CUDA extensions in oneAPI may be either `SYCL_EXT_ONEAPI_CUDA_<feature>`,
+or just `SYCL_EXT_ONEAPI_<feature>`.
+
+Use of a given CUDA extension also requires that a chosen device has the
+required compute capability to use the CUDA extension. This can be determined
+using `sycl::aspect`s. Non-core SYCL aspects may be defined by an
+implementation which would allow this check to happen at runtime.
+
+The table below shows a proposal for SYCL supported CUDA extensions. This should
+be populated by other members of the SYCL community.
+
+[[table.extensionsupport]]
+.SYCL support for CUDA 11.3 extensions
+[width="100%",options="header",cols="35%,35%,15%, 15"]
+|====
+| CUDA Extension                            | SYCL Aspect   | Feature Test Macro               | Required Compute Capability 
+|====
```

Comment:
I don't think any of this needs to be in the interop specification.  Vendors are always free to publish extensions, and each extension should describe its interface such as macros, aspects, etc.  I don't think we need any general description of this in the core backend interop specification.

I also don't think we want a list of extensions here in the core backend interop specification.  We don't want to update this backend interop spec (and get Khronos ratification) each time a vendor publishes an extension.

Reply:
Sounds sensible to me. @AerialMantis ?

Reply:
OK I will remove this.